Sort/HelperClass.java

```diff
@@ -0,0 +1,36 @@
+package Sort;
+
+public class HelperClass {
+
+    int[] swapElements(int first, int second, int[] arr) {
+        if ( first >= 0 & first <= arr.length & second >= 0 & second <= arr.length & first != second){
```

Comment:
Too complicated.
Do not chech anything - The responsibility to ensure that valid indices were passed is on the method user.
Just swap arr[first] and arr[second].